src/mmrelay/matrix/replies.py

```
@@ -0,0 +1,423 @@
+
+    Parameters:
+        text (str): Input text to truncate.
+        max_bytes (int): Maximum allowed size in bytes for the UTF-8 encoded result.
+
+    Returns:
+        str: A string whose UTF-8 encoding is at most `max_bytes` bytes.
+    """
+    truncated_text = text.encode(facade.DEFAULT_TEXT_ENCODING)[:max_bytes].decode(
+        facade.DEFAULT_TEXT_ENCODING, facade.ENCODING_ERROR_IGNORE
+    )
+    return truncated_text
```

Ruff (0.15.9)

[warning] Unnecessary assignment to `truncated_text` before `return` statement

Remove unnecessary assignment

(RET504)